### PR TITLE
FCE-883 / Bundle webrtc package in ts-client

### PR DESCRIPTION
**Pull request by czerwiukk**
## Description

Configure TSUP to bundle webrtc client in the ts client.
Additionally add `protobufs` package and store the protobufs there, instead of keeping them in `ts-client`.

## Motivation and Context

This allows to release `ts-client` without publishing `webrtc-client`.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)

### CODE REVIEW

File: packages/ts-client/src/index.ts [hunk 1]
```diff
@@ -18,6 +18,8 @@ export type { AuthErrorReason } from './auth.js';
 
 export { isAuthError, AUTH_ERROR_REASONS } from './auth.js';
 
+export { sdkVersion } from './sdkVersion';
```
Comment (mironiasty) [inline packages/ts-client/src/index.ts:21]: Why this line was added?

**Review by mironiasty — COMMENTED** [review]
How did you test that package will still work after publishing (and no files are missing?)

**Comment by czerwiukk** [comment]
> How did you test that package will still work after publishing (and no files are missing?)

@mironiasty I used `yarn pack` to create a tarball, installed it via `yarn add package.tgz` and ran the example.

File: packages/ts-client/src/index.ts (hunk 1 above)
Comment (czerwiukk) [reply packages/ts-client/src/index.ts:21]: I recreated the protos submodule. There is no tags and I forgot to pull a 0.9.0 compatible commit.
Will undo.

**Review by Karolk99 — APPROVED** [review]

**Pull request merged**